DDDEastAnglia/Controllers/HomeController.cs

```diff
@@ -9,15 +9,17 @@ namespace DDDEastAnglia.Controllers
     public class HomeController : Controller
     {
         private readonly IConferenceLoader conferenceLoader;
+        private readonly SponsorModelQuery sponsorModelQuery;
 
-        public HomeController(IConferenceLoader conferenceLoader)
+        public HomeController(IConferenceLoader conferenceLoader, SponsorModelQuery sponsorModelQuery)
```

Comment:
> Perhaps an example of something in the existing code base which does this correctly would help? Is there one?

To my knowledge, there isn't 😔 I'd like the interface to return Domain models: everything should deal in those until it has to go to an external system of some form (so, e.g., a query object maps data access results to a Domain model).